features/finder_frontend.feature

```diff
@@ -57,3 +57,8 @@ Feature: Finder Frontend
   Scenario: check that advanced search returns results
     When I visit "/search/advanced?topic=/education&group=news_and_communications"
     Then I should see filtered documents
+
+  @high
+  Scenario: check malicious code does not execute
+    When I visit "/government/organisations/hm-revenue-customs/contact?keywords=<script>alert(document.cookie);</script>"
+    Then I should see "<script>alert(document.cookie)" in the result-info class
```

Comment:
Do we need to say "in the result-info class" here?  Would be nicer to decouple the step text from the specific implementation of the page, e.g. saying "results section" (if that's what it is)